Lavinia-api/Models/Party.cs

```diff
@@ -6,12 +6,7 @@ namespace LaviniaApi.Models
     // API v1
     public class Party
     {
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public int PartyId { get; set; }
-
         [Required] public string Name { get; set; }
-
-        public string InternationalName { get; set; }
-        public string ShortName { get; set; }
+        [Required] public string Code { get; set; }
```

Comment:
Won't this break API v1?

Reply:
No, this model was apparently never used in API v1.

Reply:
Then we might want to remove the comment about API v1.

Reply:
Updated the comment in a new commit.